### DISCUSSION TITLE
Active-time based roles

### DISCUSSION THREAD

**Discussion by lorenzocorallo** [MrAugu/discord-xp#24] [Q&A]
I would like to use this package to assign XP based on the active time of a user. So if a user stays in a vocal channel (with blacklist) with audio activated earn XP and could rankup after x hours connected (like ts3-ranksystem) which gives him a new role.

**Comment by Myst82015** [comment]
> 
> 
> I would like to use this package to assign XP based on the active time of a user. So if a user stays in a vocal channel (with blacklist) with audio activated earn XP and could rankup after x hours connected (like ts3-ranksystem) which gives him a new role.


Are you asking if that is possible with this package or is this something we should add?

**Reply by lorenzocorallo** [reply]
I think it's not possible with this package so it should be added 'cause it would be a unique, fantastic and useful feature

**Reply by Myst82015** [reply]
You can do that very easily yourself. 
This won't be added anytime soon.
